tests/test_records.py

```diff
@@ -0,0 +1,66 @@
+# pylint: disable=redefined-outer-name
+"""Tests for Records resource"""
+import json
+import pytest
+import jsend
+from falcon import testing
+import service.microservice
+
+@pytest.fixture()
+def client():
+    """ client fixture """
+    return testing.TestClient(service.microservice.start_service())
+
+def test_records_welcome(client):
```

Comment:
what's the "welcome" in this?

Reply:
just basic index. renamed to reflect change. 7982ff7f070f61fc34ae63b85ee5ac49dc13095a